app/server/functions/src/main/kotlin/com/supercilex/robotscouter/server/functions/Templates.kt

```diff
@@ -50,32 +50,39 @@ fun matchTemplateMetrics() = metrics {
     header("a", "Scout info")
     text("b", "Name")
 
-    header("c", "Auto")
-    checkbox("d", "Crossed baseline")
-    selector("e", "Put cube in Switch") {
-        +Item("a", "Didn't attempt")
-        +Item("b", "Attempted")
-        +Item("c", "Successful")
-        +Item("d", "Unknown", true)
+    header("c", "Sandstorm")
+    selector("d", "Starting location") {
+        +Item("a", "HAB Level 1")
+        +Item("b", "HAB Level 2")
+        +Item("c", "Unknown", true)
     }
-    selector("f", "Put cube in Scale") {
-        +Item("a", "Didn't attempt")
-        +Item("b", "Attempted")
-        +Item("c", "Successful")
-        +Item("d", "Unknown", true)
+    checkbox("e", "Successfully crossed HAB line")
+    counter("f", "Panels on Cargo Ship")
+    counter("g", "Panels on low Rocket Hatches")
+    counter("h", "Panels on middle/high Rocket Hatches")
+    counter("i", "Cargo in Cargo Ship")
+    counter("j", "Cargo in low Rocket Bays")
+    counter("k", "Cargo in middle/high Rocket Bays")
+
+    header("l", "Teleop")
+    counter("m", "Panels on Cargo Ship")
+    counter("n", "Panels on low Rocket Hatches")
+    counter("o", "Panels on middle/high Rocket Hatches")
+    counter("p", "Cargo in Cargo Ship")
+    counter("q", "Cargo in low Rocket Bays")
+    counter("r", "Cargo in middle/high Rocket Bays")
+    stopwatch("s", "Cycle times on Panels")
+    stopwatch("t", "Cycle times on Cargo")
```

Comment:
Last nit: what about calling these "Cargo delivery cycle time"?

Reply:
I like that! Should we have "Panel delivery cycle time" instead of "Cycle times on Panels" as well?

Reply:
Yeah, for both of them.